src/tests/cli-tests/tests/asm.test.ts

```diff
@@ -0,0 +1,41 @@
+import {executeCommand} from "../src/helper";
+import { paths } from '../src/entities';
+
+
+//id1746
+describe("Run with --asm by default", () => {
+  const command = `zksolc ${paths.pathToBasicSolContract} --asm`;
+  const result = executeCommand(command);
+  const commandInvalid = 'zksolc --asm';
+  const resultInvalid = executeCommand(commandInvalid);
+
+  it("Valid command exit code = 0", () => {
+    expect(result.exitCode).toBe(0);
+  });
+
+  it("--asm output is presented", () => {
+      expect(result.output).toMatch(/(.func_begin0:)/i);
+      expect(result.output).toMatch(/(.func_end0:)/i);
+      expect(result.output).toMatch(/(CPI0_0:)/i);
```

Comment:
I believe these constants may be not present in every assembly output or they may slightly change eventually.
Let's match `__entry` as the first match instead.

Reply:
To clarify, should we match just this one special word from the whole file? (or all that inside/under this word?) Perhaps our idea is to catch such changes if any, no?

`
        .text
        .file   "contract.sol:C"
        .globl  __entry
__entry:
.func_begin0:
        and!    1, r2, r1
        jump.eq @.BB0_2
        add     128, r0, r1
        st.1    64, r1
        context.get_context_u128        r1
        sub!    r1, r0, r1
        jump.ne @.BB0_2
        add     32, r0, r1
        st.2    256, r1
        st.2    288, r0
        add     @CPI0_0[0], r0, r1
        ret.ok.to_label r1, @DEFAULT_FAR_RETURN
.BB0_2:
        add     r0, r0, r1
        ret.revert.to_label     r1, @DEFAULT_FAR_REVERT
.func_end0:

        .note.GNU-stack
        .rodata
CPI0_0:
        .cell 53919893334301279589334030174039261352344891250716429051063678533632
`

Reply:
@hedgar2017 sure, now I see the reason, I have updated this step.